skiko/buildSrc/src/main/kotlin/properties.kt

```diff
@@ -78,17 +78,17 @@ enum class SkiaBuildType(
     val msvcLinkerFlags: Array<String>
 ) {
     DEBUG(
-        "Debug",
+        id = "Debug",
         flags = arrayOf("-DSK_DEBUG"),
         clangFlags = arrayOf("-std=c++17", "-g", "-DSK_TRIVIAL_ABI=[[clang::trivial_abi]]"),
-        msvcCompilerFlags = arrayOf("/Zi /std:c++17"),
+        msvcCompilerFlags = arrayOf("/Zi", "/std:c++17"),
         msvcLinkerFlags = arrayOf("/DEBUG"),
     ),
     RELEASE(
         id = "Release",
         flags = arrayOf("-DNDEBUG"),
         clangFlags = arrayOf("-std=c++17", "-O3"),
-        msvcCompilerFlags = arrayOf("/O2 /std:c++17"),
+        msvcCompilerFlags = arrayOf("/O2", "/std:c++17"),
```

Comment:
Let's remove msvc if it is not used now (from Development.md as well)

Reply:
In fact, we still use MSVC flags, because on Windows, we use `clang-cl` and `lld-link` which are MSVC compatible compiler (cl) and linker (link) frontends, respectively. They understand the Windows .lib and .obj format, and they also use MSVC-like flags. We can rename these to something like `winCompilerFlags` and `winLinkerFlags` to make it less confusing.

Reply:
Sounds good.

Also, let's add TODO:
```
// TODO consider reusing `clangFlags` and test that it behaves the same or better:
//  - different optimisation levels (O3 instead O2 in Release)
//  - different Debug options
```

Reply:
I'm not entirely sure clang-cl supports the same flags as clang:

> To be compatible with cl.exe, clang-cl supports most of the same command-line options. Those options can start with either / or -. It also supports _some_ of Clang’s core options, such as the -W options.

I can still leave the TODO though. Theoretically the `/clang:` option can be used to pass Clang options unchanged. And the equivalent of Clang's `-O3` is `/Ot`, according to [the documentation](https://clang.llvm.org/docs/UsersManual.html#id12).

Reply:
Also note that I have [disabled the trivial ABI in Skia-pack](https://github.com/JetBrains/skia-pack/blob/3258ba9f76930af325f690dbb232261b0dbc3ffb/script/build.py#L95) to keep it compatible with MSVC, so if we decide to re-use Clang debug flags, we would have to remove that line from Skia-pack too.

Reply:
Ok, let's keep without todo